docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+networks:
+  db-data-net:
+    driver: bridge
+services:
+  pythonapp:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8081:8081"
+    environment:
+      - ENV=local
+    networks:
+      - db-data-net
+    depends_on:
+      - mysql
+    restart: unless-stopped
+  mysql:
+    image: mysql:latest
```

Comment:
Using `mysql:latest` is not recommended for production environments. Consider specifying a stable version like `mysql:5.7` or `mysql:8.0` to avoid potential compatibility issues.